UnityProject/Assets/Scripts/Objects/ObjectContainer.cs

```diff
@@ -135,13 +165,13 @@ public void StoreObjects(IEnumerable<GameObject> gameObjects)
 
 		public void GatherObjects()
 		{
-			foreach (var entity in registerTile.Matrix.Get<ObjectBehaviour>(registerTile.LocalPositionServer, true))
+			foreach (var entity in registerTile.Matrix.Get<UniversalObjectPhysics>(registerTile.LocalPositionServer, true))
 			{
 				// Don't add the container to itself...
 				if (entity.gameObject == gameObject) continue;
 
 				// Can't store secured objects (exclude this check on mobs as e.g. magboots set pushable false)
-				if (entity.TryGetComponent<HealthV2.LivingHealthMasterBase>(out _) == false && entity.IsPushable == false) continue;
+				if (entity.IsNotPushable) continue;
```

Comment:
I removed the player specific storage, and is now just one storage

Reply:
Hmm I'm not sure what you mean. So the original issue was that players with magboots enabled (which set the player as immovable) couldn't enter lockers - they'd stay outside when the locker closed.

Reply:
yeah, I think going for that approach is good and I can do it as part of rethinking space wind, I'll do that in a separate PR though since it's just a generic push currently